src/platforms/react-native/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -298,6 +230,152 @@ class CustomInstrumentation extends RoutingInstrumentation {
 }
 ```
 
+## Features
+
+Types of features:
+
+- **Measurement**: Measurements are values that we add onto any transaction running on the global Sentry scope and can be viewed on the dashboard both within Mobile Vitals and the transaction view.
+- **Span(s)**: This feature will add span(s) to any transaction running on the global Sentry scope of your app.
+
+Below are details on the automatic instrumentation features that we provide:
+
+### App Start Instrumentation
+
+_Type: **Measurement, Span**_
+
+The App Start Instrumentation provides insight into how long your application takes to launch. It tracks how long it takes the app from the **earliest native process initialization** until when the **React Native root component mounts**.
+
+<Note>
+
+If you don't [wrap your root component with Sentry](#wrap-your-root-component), the App Start measurement will finish when the JavaScript code is initialized instead of the first component mount.
+
+</Note>
+
+The SDK differentiates between a cold and a warm start, and doesn't track hot starts/resumes.
+
+- **Cold start**: App launched for the first time, after a reboot or update. The app is not in memory and no process exists.
+- **Warm start**: App launched at least once, is partially in memory, and no process exists.
+
+### Slow and Frozen Frames
+
+_Type: **Measurement**_
+
+Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the transactions you capture.
```

Comment:
```suggestion
Unresponsive UI and animation hitches annoy users and degrade the user experience. Two measurements to track these types of experiences are slow frames and frozen frames. If you want your app to run smoothly, you should try to avoid both. The SDK adds these two measurements for the transactions you capture.
```